<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskDialog</class>
 <widget class="QDialog" name="TaskDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>‰ªªÂãô</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QDialog {
      background-color: #f5f5f5;
    }
    QLabel {
      color: #2c3e50;
      font-size: 13px;
    }
    QLineEdit, QTextEdit, QDateTimeEdit, QComboBox {
      padding: 8px;
      border: 1px solid #bdc3c7;
      border-radius: 5px;
      background-color: white;
      color: #2c3e50;
      font-size: 13px;
    }
    QLineEdit:focus, QTextEdit:focus, QDateTimeEdit:focus, QComboBox:focus {
      border: 2px solid #3498db;
    }
    QLineEdit:disabled, QTextEdit:disabled, QDateTimeEdit:disabled {
      background-color: #ecf0f1;
      color: #95a5a6;
    }
    QCheckBox {
      color: #2c3e50;
      font-size: 12px;
      spacing: 8px;
    }
    QCheckBox::indicator {
      width: 18px;
      height: 18px;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <property name="spacing">
    <number>15</number>
   </property>
   
   <!-- Ê®ôÈ°å -->
   <item>
    <widget class="QLabel" name="dialogTitle">
     <property name="text">
      <string>Êñ∞Â¢û‰ªªÂãô</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 20px; font-weight: bold; color: #2c3e50; padding-bottom: 10px;</string>
     </property>
    </widget>
   </item>
   
   <!-- ‰ªªÂãôÊ®ôÈ°å -->
   <item>
    <widget class="QLabel" name="lblTitle">
     <property name="text">
      <string>Ê®ôÈ°å *</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editTitle">
     <property name="placeholderText">
      <string>Ëº∏ÂÖ•‰ªªÂãôÊ®ôÈ°å...</string>
     </property>
    </widget>
   </item>
   
   <!-- ÊèèËø∞ -->
   <item>
    <widget class="QLabel" name="lblDescription">
     <property name="text">
      <string>ÊèèËø∞</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="editDescription">
     <property name="placeholderText">
      <string>Ëº∏ÂÖ•‰ªªÂãôÊèèËø∞...</string>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   
   <!-- Êà™Ê≠¢Êó•ÊúüËàáÊèêÈÜíÊôÇÈñìÔºàÊ∞¥Âπ≥ÊéíÂàóÔºâ -->
   <item>
    <widget class="QWidget" name="dateTimeRow">
     <layout class="QHBoxLayout" name="dateTimeLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>20</number>
      </property>
      <!-- Êà™Ê≠¢Êó•Êúü -->
      <item>
       <widget class="QWidget" name="dueDateWidget">
        <layout class="QVBoxLayout" name="dueDateLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="lblDueDate">
           <property name="text">
            <string>üìÖ Êà™Ê≠¢Êó•Êúü</string>
           </property>
           <property name="styleSheet">
            <string notr="true">font-weight: bold;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="editDueDateTime">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="displayFormat">
            <string>yyyy/MM/dd HH:mm</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkNoDueDate">
           <property name="text">
            <string>ÁÑ°Êà™Ê≠¢Êó•Êúü</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <!-- ÊèêÈÜíÊôÇÈñì -->
      <item>
       <widget class="QWidget" name="reminderWidget">
        <layout class="QVBoxLayout" name="reminderLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="lblReminder">
           <property name="text">
            <string>‚è∞ ÊèêÈÜíÊôÇÈñì</string>
           </property>
           <property name="styleSheet">
            <string notr="true">font-weight: bold;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="editReminderDateTime">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="displayFormat">
            <string>yyyy/MM/dd HH:mm</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkNoReminder">
           <property name="text">
            <string>‰∏çË®≠ÂÆöÊèêÈÜí</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   
   <!-- ÂÑ™ÂÖàÁ¥öËàáÊ®ôÁ±§ÔºàÊ∞¥Âπ≥ÊéíÂàóÔºâ -->
   <item>
    <widget class="QWidget" name="priorityTagRow">
     <layout class="QHBoxLayout" name="priorityTagLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>20</number>
      </property>
      <!-- ÂÑ™ÂÖàÁ¥ö -->
      <item>
       <widget class="QWidget" name="priorityWidget">
        <layout class="QVBoxLayout" name="priorityLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="lblPriority">
           <property name="text">
            <string>‚≠ê ÂÑ™ÂÖàÁ¥ö</string>
           </property>
           <property name="styleSheet">
            <string notr="true">font-weight: bold;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboPriority">
           <item>
            <property name="text">
             <string>‰Ωé</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>‰∏≠</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>È´ò</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <!-- Ê®ôÁ±§ -->
      <item>
       <widget class="QWidget" name="tagWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="tagLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="lblTags">
           <property name="text">
            <string>üè∑Ô∏è Ê®ôÁ±§</string>
           </property>
           <property name="styleSheet">
            <string notr="true">font-weight: bold;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editTags">
           <property name="placeholderText">
            <string>Â∑•‰Ωú, Â≠∏Áøí, ÁîüÊ¥ªÔºàÈÄóËôüÂàÜÈöîÔºâ</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   
   <!-- ÈñìÈöî -->
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   
   <!-- ÊåâÈàïÂçÄ -->
   <item>
    <widget class="QWidget" name="buttonWidget">
     <layout class="QHBoxLayout" name="buttonLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <spacer name="buttonSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>ÂèñÊ∂à</string>
        </property>
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
            background-color: #95a5a6;
            color: white;
            border: none;
            padding: 10px 25px;
            font-size: 14px;
            border-radius: 5px;
          }
          QPushButton:hover {
            background-color: #7f8c8d;
          }
         </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string>ÂÑ≤Â≠ò</string>
        </property>
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
            background-color: #3498db;
            color: white;
            border: none;
            padding: 10px 25px;
            font-size: 14px;
            border-radius: 5px;
          }
          QPushButton:hover {
            background-color: #2980b9;
          }
         </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>TaskDialog</receiver>
   <slot>reject()</slot>
  </connection>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>TaskDialog</receiver>
   <slot>accept()</slot>
  </connection>
 </connections>
</ui>
